public/apps/configuration/utils/display-utils.tsx

```diff
@@ -62,7 +62,7 @@ export function renderCustomization(reserved: boolean) {
         <EuiIcon type={reserved ? 'lock' : 'pencil'} />
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
-        <EuiText size="xs">{reserved ? 'Reserved' : 'Custom'}</EuiText>
+        <EuiText className="table-items">{reserved ? 'Reserved' : 'Custom'}</EuiText>
```

Comment:
In other files (e.g user-list.tsx) which are specific to a component, there is no concern. However, for this util class, the common function may be called in a non-table context. I think we can either pass class name to it or have the class name on the parent level where this is being used.

Reply:
I have checked the code and right now we are not using any function that have changed in this PR in a non-table context. Table uses `small` as default text size and `EuiText` component uses `medium` as default text size. So if set the class name on the parent level, it will not change the font size of `EuiText`. Also it would be easy if we can set font-size at parent level, so that child component in display-utils will inherit the font-size from parent component.

Reply:
In this case, I would suggest passing the class name as parameter to this function instead of restricting the function to be tired with table context.

Reply:
Acknowledged. Passing class name as parameter to display util function.